src/main/resources/default/taglib/w/jobs.html.pasta

```diff
@@ -1,11 +1,12 @@
 <i:arg type="String" name="uri" default="@call.getRequest().uri()"/>
 <i:arg type="Object" name="target"/>
 
-<i:if test="jobsService != null">
-    <i:local name="jobs" value="@jobsService.getMatchingInteractiveJobs(uri, target)"/>
-    <i:if test="!jobs.isEmpty()">
+<i:local name="jobs" value="Injector.context().getPart(Jobs.class)" />
```

Comment:
Dafür gibts doch das PartMacro?

Reply:
nope PartMacro uses findPart which throws in case if a missing part. As the Jobs.class can be disabled via the framework config, we have to deal with this and use getPart here...